routes/web.php

```diff
@@ -23,10 +23,14 @@
 Route::middleware("auth")->group(function (): void {
     Route::post("/logout", [LogoutController::class, "logout"])->name("logout");
     Route::get("/dashboard", fn(): Response => inertia("Dashboard"))->name("dashboard");
+
+    Route::middleware(["role:admin"])->group(function (): void {
```

Comment:
I'm not sure if this middleware "role:admin" should be like this?

Reply:
Why not?

Reply:
I mean that this middleware for now it's okay. But when you will have permissions for each action, it should change.

Reply:
We generally use gates/policy for stuff like this and not on middleware level. Also you can use permissions.